diff -ruN reg_tmp/system.reg reg_tmp_fixed/system.reg
--- reg_tmp/system.reg	2018-04-03 13:53:26.000000000 +0800
+++ reg_tmp_fixed/system.reg	2019-04-03 21:34:07.000000000 +0800
@@ -30955,15 +30955,16 @@
 [Software\\Microsoft\\Windows\\CurrentVersion\\Winlogon] 1486457887
 #time=1d281204c9b71e0
 
-[Software\\Microsoft\\Windows NT\\CurrentVersion] 1507795882
-#time=1d34331b1382cd2
-"CSDVersion"="Service Pack 3"
-"CurrentBuildNumber"="2600"
+[Software\\Microsoft\\Windows NT\\CurrentVersion] 1553355758
+#time=1d4e18f0af44402
+"CSDVersion"=""
+"CurrentBuild"="17134"
+"CurrentBuildNumber"="17134"
 "CurrentType"="Uniprocessor Free"
-"CurrentVersion"="5.1"
+"CurrentVersion"="10.0"
 "InstallDate"=dword:4be5019a
 "ProductId"="12345-oem-0000001-54321"
-"ProductName"="Microsoft Windows XP"
+"ProductName"="Microsoft Windows 10"
 "RegisteredOrganization"=""
 "RegisteredOwner"=""
 "SystemRoot"="C:\\windows"
diff -ruN reg_tmp/update.policy reg_tmp_fixed/update.policy
--- reg_tmp/update.policy	2018-04-03 13:53:26.000000000 +0800
+++ reg_tmp_fixed/update.policy	2018-02-23 20:31:32.000000000 +0800
@@ -8,15 +8,14 @@
 ;; Ingore     - Ingore file when copy.
 
 ;; Ingore update.policy itself.
-;; "^update.policy$" = "Ingore"
+"^update.policy$" = "Ingore"
 "^.update-timestamp$" = "Ingore"
 
 ;; System registry files.
 "^(system|userdef|user).reg$" = "Registry"
 
 ;; Addtional assist files.
-"^drive_c/deepin" = "Keep"
-"^drive_c/deepin/EnvInit.sh" = "Keep"
+"^drive_c/Deepin" = "Keep"
 
 ;; Microsoft Office files.
 "^drive_c/Windows/(?:control\.ini|hh\.dat|mapiuid\.ini|Reg Save Log\.txt)$" = "Override"
diff -ruN reg_tmp/user.reg reg_tmp_fixed/user.reg
--- reg_tmp/user.reg	2018-04-03 13:53:26.000000000 +0800
+++ reg_tmp_fixed/user.reg	2019-04-03 21:35:35.000000000 +0800
@@ -815,6 +815,7 @@
 "androidassist"=""
 "mlang"="native,builtin"
 "msls31"="native,builtin"
+"ntoskrnl.exe"=""
 "riched20"="native,builtin"
 "txplatform.exe"=""
 "txupd.exe"=""
