diff -ruN reg_tmp/system.reg reg_tmp_fixed/system.reg
--- reg_tmp/system.reg	2019-08-27 15:06:17.000000000 +0800
+++ reg_tmp_fixed/system.reg	2020-03-10 11:14:33.397830695 +0800
@@ -30982,15 +30982,16 @@
 [Software\\Microsoft\\Windows\\CurrentVersion\\Winlogon] 1486457887
 #time=1d281204c9b71e0
 
-[Software\\Microsoft\\Windows NT\\CurrentVersion] 1566803037
-#time=1d55bdc6e529798
-"CSDVersion"="Service Pack 3"
-"CurrentBuildNumber"="2600"
+[Software\\Microsoft\\Windows NT\\CurrentVersion] 1571657811
+#time=1d58803d4d74e20
+"CSDVersion"="Service Pack 1"
+"CurrentBuild"="7601"
+"CurrentBuildNumber"="7601"
 "CurrentType"="Uniprocessor Free"
-"CurrentVersion"="5.1"
+"CurrentVersion"="6.1"
 "InstallDate"=dword:4be5019a
 "ProductId"="12345-oem-0000001-54321"
-"ProductName"="Microsoft Windows XP"
+"ProductName"="Microsoft Windows 7"
 "RegisteredOrganization"=""
 "RegisteredOwner"=""
 "SystemRoot"="C:\\windows"
